### PR TITLE
chore(statics): ungated zeta and zetaevm for singapore

### PR DESCRIPTION
**Pull request by hrishikeshjain**
TICKET: COIN-2044

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -932,7 +934,7 @@ export const coins = CoinMap.fromCoins([
     18,
     UnderlyingAsset.ZETA,
     BaseUnit.ZETA,
-    COSMOS_SIDECHAIN_FEATURES
+    ZETA_FEATURES
```
[the remark this answers is not in the document]
Comment (hrishikeshjain) [reply modules/statics/src/coins.ts:937]: For other coins example sol, bera naming is `SOL_FEATURES`, `BERA_FEATURES` so naming `ZETA_FEATURES` would maintain consistency

[the remark this answers is not in the document]
Comment (rakeshgsharma) [reply modules/statics/src/coins.ts:937]: makes sense, i didnt see the complete file

**Review by rakeshgsharma — DISMISSED** [review]
LGTM

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -8583,7 +8585,8 @@ export const coins = CoinMap.fromCoins([
     'Zeta (ERC20)',
     18,
     '0xf091867ec603a6628ed83d274e835539d82e9cc8',
-    UnderlyingAsset.ZETAEVM
+    UnderlyingAsset.ZETAEVM,
+    ZETAEVM_FEATURES
```
Comment (adarsh312) [inline modules/statics/src/coins.ts:8589]: can we have the variable named as ZETA_EVM_FEATURES? its better to read

Comment (hrishikeshjain) [reply modules/statics/src/coins.ts]: done

**Review by mohammadalfaiyazbitgo — APPROVED** [review]

**Pull request merged**